src/komodo_validation011.h

```diff
@@ -100,23 +102,27 @@ int32_t gettxout_scriptPubKey(int32_t height,uint8_t *scriptPubKey,int32_t maxsi
             if ( it != pwallet->mapWallet.end() )
             {
                 const CWalletTx& wtx = it->second;
-                txref = wtx.tx;
+                tx = *wtx.tx;
                 fprintf(stderr,"found tx in wallet\n");
             }
         }
     }
 
-    if ( txref != 0 && n >= 0 && n < (int32_t)txref->vout.size() )
+    if ( &tx != 0 && n >= 0 && n < (int32_t)tx.vout.size() )
```

Comment:
&tx will always be != 0, this is the address of a local variable.   Are you sure this is what you what to do here (and above)?

Reply:
This was ported from https://github.com/jl777/chips3/blob/dev/src/komodo_validation015.h#L108 , but since we have a different BTC internals, we have to use CTransaction vs CTransactionRef, which changes some stuff

Reply:
Yeah, this needs to be changed to say "Is this CTransaction empty/null?" instead